docs/tutorials/portal/301/portal-301-5.rst

```diff
@@ -22,6 +22,246 @@ Rubin staff will respond to all questions posted there.
 
 ----
 
-**1. Log in to the Portal Aspect of the RSP.**
+1. Introduction
+===============
 
-*Tutorial coming soon!*
+This tutorial explores the LSSTComCam observations for the Euclid Deep Field South or "EDFS" field, including magnitude limits, visit distribution with time, data quality, and the distributions of stars and galaxies in color-magnitude and color-color diagrams.
+
+The **Euclid Deep Field South** covers approximately **28.1 square degrees** in the southern constellation of **Horologium**, named after the pendulum clock.
```

Comment:
People are going to mistake this for thinking the DP1 EDFS covers 28.1 sq deg when it doesn't. Reword to say something like "the full euclid field is 28.1 but the dp1 field is a small subset of X sq deg" or can just remove the sentence.